spec/test-expect.js

```diff
@@ -19,6 +19,7 @@ describes.endtoend('AMP carousel', {
     await expect(2+2).to.be.above(3);
     await expect(2+2).to.be.below(5);
 
+    await expect({a: '2', b: '3'}).to.include({a: '2'});
```

Comment:
Can you add a similar test that uses a `ControllerPromise` result? Like the tests below in `should work with async values`